cpp/oneapi/dal/table/detail/table_iface.hpp

```diff
@@ -106,6 +116,10 @@ class table_template : public Iface,
     pull_column_iface* get_pull_column_iface() override {
         return this;
     }
+
+    pull_sparse_block_iface* get_pull_sparse_block_iface() override {
```

Comment:
Why every table type shall implement this method?

Reply:
I suggest to discuss it offline, I came across on some problems